tests/internal/test_metrics.py

```diff
@@ -0,0 +1,33 @@
+from contextlib import contextmanager
+
+import mock
+
+from ddtrace.internal.metrics import Metrics
+
+
+@contextmanager
+def mock_metrics():
+    metrics = Metrics()
+    try:
+        metrics._client = mock.Mock()
+        metrics.enable()
+        yield metrics
+    finally:
+        metrics.disable()
+
+
+def test_metrics_names():
```

Comment:
Can you add a test with the namespace set?

Reply:
Sure, but the effect of that is "hidden" inside the dogstatsd library and so won't be detectable from the tests without further internal knowledge of the dependency.